web/tools/manifest.json

```diff
@@ -1,9 +1,9 @@
 {
```

Comment:
Did you check whether there are other properties a manifest.json should include?

Reply:
I think this is enough and checked the docs, but maybe I miss something.